lib/node_modules/@stdlib/array/fixed-endian-factory/lib/main.js

```diff
@@ -514,6 +514,37 @@ function factory( dtype ) { // eslint-disable-line max-lines-per-function, stdli
 		return this._buffer.byteLength;
 	});
 
+	/**
```

Comment:
This needs to be moved to before `forEach`. Currently, this is not inserted in alphabetical order.